api/v1/usage_server.proto

```diff
@@ -28,6 +28,56 @@ message AggregatedSummary {
   repeated Summary method_summaries = 2;
 }
 
+message GetUsageDataRequest {
+  string tenant_id = 1;
```

Comment:
Let's remove this. Once we provide the externally visible endpoint, tenant_id will be hidden from end users.